src/main/kotlin/io/github/mojira/arisa/modules/CrashInfoModule.kt

```diff
@@ -0,0 +1,92 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import arrow.core.left
+import arrow.core.right
+import arrow.syntax.function.partially2
+import io.github.mojira.arisa.domain.Issue
+import io.github.mojira.arisa.infrastructure.AttachmentUtils
+import com.urielsalis.mccrashlib.Crash
+import com.urielsalis.mccrashlib.CrashReader
+import com.urielsalis.mccrashlib.deobfuscator.getSafeChildPath
+import java.nio.file.Files
+import java.time.Instant
+
+class CrashInfoModule(
+    private val crashReportExtensions: List<String>,
+    private val crashReader: CrashReader
+) : Module {
+    override fun invoke(issue: Issue, lastRun: Instant): Either<ModuleError, ModuleResponse> = with(issue) {
+        Either.fx {
+            val crashes = AttachmentUtils(crashReportExtensions, crashReader).extractCrashesFromAttachments(issue)
+            val newCrashes = assertContainsNewCrash(crashes, lastRun).bind()
+            uploadDeobfuscatedCrashes(issue, newCrashes)
+
+            newCrashes
+                .filter { it.second is Crash.Minecraft }
+                .filterNot { it.first.name.endsWith("deobfuscated.txt") }
```

Comment:
We should probably restrict this to crash reports attached by the reporter (or by a helper / moderator?).
Otherwise users can vandalize existing issues by attaching unrelated crash reports, or files which Arisa considers to be crash reports.